R/utils-html_manipulation.R

```diff
@@ -378,3 +378,32 @@ html_unescape <- function(html) {
     }
   )
 }
+
```

Comment:
Add a new function that calls `glue` with `.envir = emptyenv()`, so that you are forced to pass in any inputs explicitly.

Reply:
I added `glue_explicit()`:

```r
glue_explicit <- function(...) {

  glue::glue(..., .envir = emptyenv()) %>% as.character()
}
```